imagery/i.maxlik/main.c

```diff
@@ -141,7 +141,7 @@ int main(int argc, char *argv[])
     Rast_free_cats(&cats);
 
     if (reject_fd > 0) {
-	char title[100];
+	char title[GNAME_MAX + 30];
```

Comment:
A name is not stored in title, thus GNAME_MAX seems not appropriate.

How about
```
char *title = NULL;
...
G_asprintf(&title, "Rejection Probability for %s", class_name);
```
?